WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift

```diff
@@ -157,7 +162,12 @@ private class QuickStartChecklistDataSource: NSObject, UITableViewDataSource {
 
         switch section {
         case .congratulations:
-            return shouldShowCongratulations() ? 1 : 0
+            if shouldShowCongratulations() {
+                WPAnalytics.track(.quickStartCongratulationsViewed)
```

Comment:
This is re-triggered anytime the table view is refreshed.  You can test it by changing orientation.  Is a way to bump it just one time?